diff -urN aa/bug-buddy-2.8.0/docs/C/bug-buddy-C.omf bug-buddy-2.8.0/docs/C/bug-buddy-C.omf
--- aa/bug-buddy-2.8.0/docs/C/bug-buddy-C.omf	2004-09-12 05:57:15.000000000 +0200
+++ bug-buddy-2.8.0/docs/C/bug-buddy-C.omf	2004-12-02 00:17:59.116003760 +0100
@@ -3,8 +3,8 @@
   <resource>
     <creator>baudais@kkpsi.org (Eric Baudais)</creator>
     <creator>telsa@linuxchix.org (Telsa Gwynne)</creator>
-    <contributor>kevin@kevindumpscore.com (Kevin Conder)</contributor>
     <maintainer>docs@gnome.org (GNOME Documentation Project)</maintainer>
+    <contributor>kevin@kevindumpscore.com (Kevin Conder)</contributor>
     <title>Bug Buddy Manual</title>
     <date>September 11, 2004</date>
     <version identifier="2.8" date="2004-09-11" description="Updated to bug buddy 2.8"/>
