diff -urN bug-buddy-2.16.1/src/bug-buddy.desktop.in.in bug-buddy-2.16.1.new/src/bug-buddy.desktop.in.in
--- bug-buddy-2.16.1/src/bug-buddy.desktop.in.in	2006-11-28 01:33:00.000000000 +0100
+++ bug-buddy-2.16.1.new/src/bug-buddy.desktop.in.in	2006-12-24 14:55:20.000000000 +0100
@@ -6,10 +6,11 @@
 Icon=bug-buddy
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Application;Development;
+Categories=GTK;GNOME;Development;
 StartupNotify=true
 NoDisplay=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=bug-buddy
 X-GNOME-Bugzilla-Component=general
 X-GNOME-Bugzilla-Version=@VERSION@
+# vi: encoding=utf-8
