diff -urN bug-buddy-2.8.0.orig/src/bug-buddy.desktop.in.in bug-buddy-2.8.0/src/bug-buddy.desktop.in.in
--- bug-buddy-2.8.0.orig/src/bug-buddy.desktop.in.in	2004-11-30 14:45:20.269965376 +0100
+++ bug-buddy-2.8.0/src/bug-buddy.desktop.in.in	2004-11-30 14:47:59.111817744 +0100
@@ -6,7 +6,7 @@
 Icon=bug-buddy.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Development;
+Categories=GTK;GNOME;Development;
 StartupNotify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=bug-buddy
